xarray/core/accessor_dt.py

```diff
@@ -439,7 +440,7 @@ def weekofyear(self) -> DataArray:
 
         warnings.warn(
             "dt.weekofyear and dt.week have been deprecated. Please use "
-            "dt.isocalendar().week instead.",
+            + "dt.isocalendar().week instead.",
```

Comment:
```suggestion
            "dt.isocalendar().week instead.",
```

Was this an auto-formatter?

It's not required in python; it's also a (vv small) performance drag

Reply:
Not a linter, habit as PyLint flags it